### ISSUE TITLE
[🚀 Feature]: Selenium Manager cache browser

### ISSUE THREAD

**Issue by titusfortner** [SeleniumHQ/selenium#11354]
### Feature and motivation

Improve performance by allowing a user to opt-in to the browser-ttl value. Would want both a clear and a set.

Bindings would implement this as part of the `getBinary()` step. Since this method is singleton/static, it should only run once per process. So we'd want to clear the browser cache so that it will check it first run, then be able to set browser cache to a value that should last through the execution time. 

### Usage example

```
selenium-manager --chrome --clear-browser-cache 60
```
and
```
selenium-manager --chrome --set-browser-cache 60
```

**Comment by bonigarcia** [comment]
I am lost with this feature. Is it still required?

**Comment by titusfortner** [comment]
The idea is that instead of using ttl, to cache things based on the program execution.

For instance, every time I run `mvn clean test`, I want it to check the browser version on the system, and match it to the driver version and download things as necessary, not just use what is cached.

But, if I'm starting 1000 sessions during that execution, I don't need it to check those things every time. So if there were a way to tell SM to skip all the checks and just use the latest matching in cache that would be nice. We could track "firstRun" and do an update then use cache until the program exits.

But I'm not sure how we are using ttl right now to know how these would work together.

**Comment by bonigarcia** [comment]
The TTL in Selenium Manager is inspired by the TTL for DNS, which is a well-known mechanism that refers to how long some values are cached before they are automatically refreshed.

In the case of Selenium Manager, these values are the versions found by making network requests (e.g., for chromedriver versions and, currently, for CfT versions). It is already working at does not require any further change. By default, the TTL is 1 hour and can be tuned by using configuration values, or disabled simply changing these configuration values to 0.

In summary, in my opinion, this issue is not required.

**Comment by titusfortner** [comment]
We had a big argument about browser TTL vs driver TTL, and I thought we ended up with the default for the browser TTL of 0 and a driver TTL of 24 hours. Did we change those?

Browser/driver versions online do not change that often, so only checking for new versions every 24 hours makes sense to me.

The browser version on the system can change whenever a user wants to change it, and executing a process call is very inexpensive, so it should always be checked to make sure that the right driver is always provided. 

This issue was intended as an alternative to setting a browser TTL if we are concerned about the performance of making process calls on the system. If we don't care about that, then yes, we can close this.

**Comment by bonigarcia** [comment]
Yes, there have been two changes related to TTL recently.

The first one is explained in PR #12353. Now`TTL_browsers` refers to the browser version discovered by making network requests to CfT, not by finding browser versions by doing shell commands. By default, this value now is 1 hour.

The second one is explained in #12394. Now, `TTL_drivers` is 1 hour and not 1 day since we detected that the rate change of chromedriver is quicker than it used to be. 

My next proposal will be to use a single configuration value (simply called `TTL`) for both aspects since now they refer to the same thing. That will simplify a bit the explanation of the concept of TTL in the Selenium Manager page, which is also important for people to understand this configuration key.

Overall, yes, I think we can safely close this issue.

**Comment by titusfortner** [comment]
Thanks. Also, combining TTL would be good, I think.
